### DISCUSSION TITLE
RTL-SDR V4 not working

### DISCUSSION THREAD

**Discussion by remsiux** [rtl-airband/RTLSDR-Airband#439] [Q&A] [answered]
I tried upgrading to RTL-SDR V4 but that does not seam to work. I am running the V4 on Rpi3. The scanner works with no errors but I do not get any signals on any frequency. 
When I plug in RTLSDR V3 on the same setup it works no problem.

Am I missing any drivers or setup?

**Comment by charlie-foxtrot** [comment]
If rtl-airband runs then it is talking to some device. What does lsusb show when the v3 is connected vs the v4?  Are you using the same gain settings on both?  Have you tried comparing the waterfalls between the two?

**Reply by remsiux** [reply]
In lsusb -v i see that 'iManufacturer' and 'iProduct' is different for both devices.
For V3 it is 'Realtek' and 'RTL2838UHIDIR'
For V4 'RTLSDRBlog' and 'Blog V4'

When I try them I just unplug one device and plug the other.
On windows with same gain waterfalls seem near identical for both.

**Comment by charlie-foxtrot** [comment]
Yes, according to this new drivers are required: https://www.rtl-sdr.com/V4/

**Comment by remsiux** [comment]
Now I get errors trying to make the program. Not too familiar with linux to know how to solve that.

"
make[2]: *** No rule to make target '/usr/lib/arm-linux-gnueabihf/librtlsdr.so', needed by 'src/rtl_airband'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:115: src/CMakeFiles/rtl_airband.dir/all] Error 2
make: *** [Makefile:149: all] Error 2
"

**Comment by remsiux** [comment, marked as answer]
Ahh, got it working. Needed to use "Alternative Debian Package Installation Method"

https://www.rtl-sdr.com/V4/
